### PR TITLE
lacework-vulnerability-scanner v0.23.2 (new cask)

### PR DESCRIPTION
**Pull request by phrabec**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/l/lacework-vulnerability-scanner.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cask "lacework-vulnerability-scanner" do
+  version "0.23.2"
+  sha256 "11f1617e9eb033e9ccdfcdfefcb4fc2fdc48944fd7d013da2915bb72e1ef1e83"
+
+  url "https://github.com/lacework/lacework-vulnerability-scanner/releases/download/v#{version}/lw-scanner-darwin-amd64",
+      verified: "github.com/lacework/lacework-vulnerability-scanner/"
```
[the remark this answers is not in the document]
Comment (phrabec) [reply Casks/l/lacework-vulnerability-scanner.rb:6]: Unfortunately no, only the AMD64 version is available. More about available variants: https://github.com/lacework/lacework-vulnerability-scanner/releases

[the remark this answers is not in the document]
Comment (krehel) [reply Casks/l/lacework-vulnerability-scanner.rb:6]: Yeah, and this is not a universal binary either. (just for reference, from an M1)

```
 > ./lw-scanner-darwin-amd64
zsh: bad CPU type in executable: ./lw-scanner-darwin-amd64
```

**Comment by krehel** [comment]
Hey @Homebrew/cask -

It appears this binary is not signed.  The audit for detecting bare binaries as unsigned is currently not working.  I'm trying to see what can be done to fix the issue.

IMHO if we are adhering to our standards of unsigned apps, that should extend the same.

**Comment by miccal** [comment]
Thank you @phrabec.

**Review by miccal — APPROVED** [review]

**Comment by bevanjkay** [comment]
If there is no universal binary here should it also depends on intel? Or require rosetta?

**Pull request merged**